scripts/contractInteraction/loan_tokens.py

```diff
@@ -855,4 +855,74 @@ def replaceLoanTokenSettingsLowerAdmin():
     loanTokenLogicBeaconWrbtc = Contract.from_abi("LoanTokenLogicBeacon", address=conf.contracts['LoanTokenLogicBeaconWrbtc'], abi=LoanTokenLogicBeacon.abi, owner=conf.acct)
     print("Registering Loan Protocol Settings Module to LoanTOkenLogicBeaconWrbtc")
     data = loanTokenLogicBeaconWrbtc.registerLoanTokenModule.encode_input(loanTokenSettingsLowerAdmin.address)
-    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconWrbtc.address, data, conf.acct)
\ No newline at end of file
+    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconWrbtc.address, data, conf.acct)
+
+def transferBeaconOwnershipToGovernance():
+    # transfer beacon LM
+    print("Transferring beacon LM ownserhip to: ", conf.contracts['TimelockOwner'])
+    loanTokenLogicBeaconLM = Contract.from_abi("loanTokenLogicBeaconLM", address=conf.contracts['LoanTokenLogicBeaconLM'], abi=LoanTokenLogicBeacon.abi, owner=conf.acct)
+    data = loanTokenLogicBeaconLM.transferOwnership.encode_input(conf.contracts['TimelockOwner'])
+    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconLM.address, data, conf.acct)
+
+    # transfer beacon wrbtc
+    print("Transferring beacon WRBTC ownserhip to: ", conf.contracts['TimelockOwner'])
+    loanTokenLogicBeaconWrbtc = Contract.from_abi("loanTokenLogicBeaconWrbtc", address=conf.contracts['LoanTokenLogicBeaconWrbtc'], abi=LoanTokenLogicBeacon.abi, owner=conf.acct)
+    data = loanTokenLogicBeaconWrbtc.transferOwnership.encode_input(conf.contracts['TimelockOwner'])
+    sendWithMultisig(conf.contracts['multisig'], loanTokenLogicBeaconWrbtc.address, data, conf.acct)
+
+def transferLoanTokenOwnershipToGovernance():
```

Comment:
can we split this into 2 functions?
the first function should transfer the admin role
the second function should transfer the ownership.
else, I see the problem that people are signing, one tx fails for whichever reason, and we end up with an admin not properly set and require a SIP to change it.

also: could you verify that the exchequer keeps pausing rights on all of these?

Reply:
Yes, as long as we did not transfer the pauser role, the exchequer will be the one who has the pausing rights for all of the loan tokens